### ISSUE TITLE
Design rich editor alt text controls

### ISSUE THREAD

**Issue by tburry** [vanilla/vanilla#8506]
- Images in the editor display the image whether they are uploaded or linked.
- There should be a button hovering over the image to add alt text. The button should appear when the image is selected. You can illustrate this by showing the blue border (see the current site for the width of the current size/color of the border).
- Clicking the button will reveal a modal or dropdown similar to the other controls within the editor.
- The modal will have one field:
    - Label: Alt Text
    - Description: You can give the image alternative text to help users with accessibility concerns and improve SEO.
- Mockups should include a case where an image that is smaller than the button is uploaded.

**Comment by ElisaAlmeida** [comment]
It's done. 
Mockup Desktop:
https://app.zeplin.io/project/5c65db78a981283c0b8d956e/screen/5c92a20540d00405f5b54dea
Mockups Mobile:
https://app.zeplin.io/project/5c65db78a981283c0b8d956e/screen/5c93a78a4e90c024d1dac713
https://app.zeplin.io/project/5c65db78a981283c0b8d956e/screen/5c93a78ad12e87057b33637c
